assets/src/editor/events/ticket-assignments-manager/with-ticket-assignments-manager-modal.js

```diff
@@ -19,61 +19,42 @@ import TicketAssignmentsManagerModal from './ticket-assignments-manager-modal';
  *
  * @return {function} A composed component
  */
-const WithTicketAssignmentsManagerModal = (
+const withTicketAssignmentsManagerModal = (
 	modalPropsCallback = () => null
 ) => createHigherOrderComponent(
-	( OriginalComponent ) => {
-		return class extends Component {
-			constructor( props ) {
-				super( props );
-				this.state = { showManager: false };
+	( WrappedComponent ) => ( props ) => {
+		const {
+			dateEntity,
+			ticketEntity,
+		} = props;
+		const [ showManager, setShowManager ] = useState( false );
+		const toggleManager = useCallback( ( event ) => {
+			if ( event ) {
+				event.preventDefault();
+				event.stopPropagation();
```

Comment:
I converted this to a functional component because I think the logic is a bit clearer and we can take advantage of React hooks here.